File: PRAKTIKUM MODUL 1 C DAN PY/BAHASA C/PRAK1010-2610817210007-MuhammadZakiMubarak.c
#include <stdio.h>
#include <math.h>

int main(void) {
    int base = 5;
    int height = 12;

    int slant = (int)sqrt((base * base) + (height * height));
    
    int circumference = base + height + slant;
    int area = (base * height) / 2;

    printf("diketahui :\n");
    printf("alas = %d cm\n", base);
    printf("tinggi = %d cm\n\n", height);

    printf("Jawab : \n");
    printf("Sisi A = %d cm\n", base);
    printf("Sisi B = %d cm\n", height);
    printf("Sisi C = %d cm\n", slant);
    
    printf(" = %d cm\n", circumference);
    printf("luas = %d cm\n", area);

    return 0;
}

File: PRAKTIKUM MODUL 1 C DAN PY/PYTHON/PRAK1010-2610817210007-MuhammadZakiMubarak..py
import math

base = 5
height = 12
slant = math.sqrt ((base * base) + (height * height))
circumference = base + height + slant
area = (base * height) / 2

print("diketahui : ")
print("alas = %d" % base)
print("tinggi = %d" % height)

print("\nJawab : ")
print("Sisi A = %d" % base)
print("Sisi B = %d" % height)
print("Sisi c = %d" % slant)

print("keliling = %d" % circumference)
print("luas = %d" % area)
